assets/dev/scss/editor/_global.scss

```diff
@@ -166,3 +165,31 @@
 i.eicon-tilted {
 	transform: rotate(90deg);
 }
+
+.elementor-control .elementor-control-content.elementor-control-media-size {
```

Comment:
should it be here or in some `controls/_media.scss` file?

Reply:
There is no such SCSS file that directly related to media control.

Reply:
Done